tests/test_fetcher.py

```diff
@@ -29,30 +22,15 @@ def test_start_build(self, mock_runtests, mock_docker_stop, mock_docker_start):
         self.assertTrue(mock_docker_start.called)
         self.assertTrue(mock_docker_stop.called)
 
-    @patch('frigg_worker.fetcher.config', mock_config)
     @responses.activate
     def test_fetch_task(self):
 
         responses.add(
             responses.GET,
-            mock_config('DISPATCHER_URL'),
+            'http://example.com',
```

Comment:
Yes, read line 35, it is hard coded there as well. It is like hard-coding in `assertEquals`